module/os/windows/zfs/zfs_windows_zvol_scsi.c

```diff
@@ -966,6 +966,8 @@ wzvol_WkRtn(__in PVOID pWkParms)
 	zfs_uio_iovec_init(&uio, &iov, 1, 0, UIO_SYSSPACE, pSrb->DataTransferLength, 0);
```

Comment:
Ahh oops, yes that would mess things up.  I think argument 4 to `zfs_uio_iovec_init` is offset in this case, so we can probably set it here. Thx